modules/cnv-about-collecting-cnv-data.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-collecting-cnv-data.adoc
+
+[id="cnv-about-collecting-cnv-data_{context}"]
+= About collecting {CNVProductName} data
+
+You can use the `oc adm must-gather` CLI command to collect information about your
+cluster, including features and objects associated with {CNVProductName}:
+
+* The Hyperconverged Cluster Operator namespaces (and child objects)
+* All namespaces (and their child objects) that belong to any {CNVProductName}
+resources
+* All {CNVProductName} Custom Resource Definitions (CRDs)
+* All namespaces that contain virtual machines
+* All virtual machine definitions
+
+To collect {CNVProductName} data with `must-gather`, you must specify the
+{CNVProductName} image:
+`--image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8`.
```

Comment:
This image is invalid/doesn't exist. It causes the command to fail with this error:
`error: gather did not start for pod must-gather-f9tvl: unable to pull image: ErrImagePull: rpc error: code = Unknown desc = Error reading manifest latest in registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8: unknown: Not Found`

Reply:
@yossisegev images weren't released yet, therefore it is expected for it to fail.
if you wish to test the tool you can use d/s builds, i.e. `brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/container-native-virtualization/cnv-must-gather:v2.1.0-12` (or other names as listed on https://brewweb.engineering.redhat.com/brew/packageinfo?packageID=72295)

Reply:
@yossisegev images weren't released yet, therefore it is expected for it to fail.
if you wish to test the tool you can use d/s builds, i.e. brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/container-native-virtualization/cnv-must-gather:v2.1.0-12 (or other names as listed on https://brewweb.engineering.redhat.com/brew/packageinfo?packageID=72295)

Reply:
@ousleyp @masayag Very well, then.
I tested the d/s image, just to verify that all the data which is detailed in the manual, does exist in the must-gather output directory.
What is left here is to verify that using the official cnv-must-gather image, i.e. **the one which is specified here in the manual** (registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8) does produce the expected CNV must-gather output. If you wish - let me know once the image is ready, and I'll verify that.